### PR TITLE
[redfish] Create default Redfish pollers and EMC OEM pollers in discovery

### PR DESCRIPTION
**Pull request by jlongever**
@RackHD/corecommitters @uppalk1 @zyoung51 @tannoa2 @keedya @geoff-reid

### CODE REVIEW

**Comment by jlongever** [comment]
test this please

File: lib/graphs/redfish-discovery-graph.js [hunk 1]
```diff
@@ -36,14 +84,47 @@ module.exports = {
...
-            'x-description': 'Set the final graph state to success when cataloging is skipped',
-            label: 'noop',
-            taskName: 'Task.noop',
-            waitOn: {
-                'when-catalog-emc': 'failed'
-            }
+        {
+            'x-description': 'Set the final graph state to success when cataloging is skipped',
+            label: 'noop',
+            taskName: 'Task.noop',
```
Comment (yyscamper) [inline lib/graphs/redfish-discovery-graph.js]: Why insert a noop task here?

**Comment by yyscamper** [comment]
:+1: after clarify the noop-task

File: lib/graphs/redfish-discovery-graph.js (hunk 1 above)
Comment (zyoung51) [inline lib/graphs/redfish-discovery-graph.js]: The reason is on mentioned on line 121: 'x-description': 'Set the final graph state to success when cataloging is skipped',

**Pull request merged**